src/main/java/io/molnarsandor/trelloclone/project/model/ProjectEntity.java

```diff
@@ -0,0 +1,49 @@
+package io.molnarsandor.trelloclone.project.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import io.molnarsandor.trelloclone.collaborator.model.CollaboratorEntity;
+import io.molnarsandor.trelloclone.project_task.model.BacklogEntity;
+import io.molnarsandor.trelloclone.user.model.UserEntity;
+import io.molnarsandor.trelloclone.util.EntitySuperClass;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@SuppressFBWarnings({"EI_EXPOSE_REP2", "EI_EXPOSE_REP"})
+@Entity
+@Table(name = "project")
+@Data
```

Comment:
Entitásokon a `@Data` annotáció használata elég "veszélyes", de persze nem annyira nagy baj, a probléma az, hogy az entitások és a sima POJO osztályok contractjai sajnos nem teljesen egyeznek meg, itt az `equals `és `hashCode `metódusokra gondolok. 

Jobban kifejtve ebben a cikkben találod meg az okokat: https://vladmihalcea.com/the-best-way-to-implement-equals-hashcode-and-tostring-with-jpa-and-hibernate/